app/views/profile/subject-knowledge.njk

```diff
@@ -18,16 +18,16 @@
 {% endblock %}
 
 {% block primary %}
-  <p class="govuk-body-l">Use this section to give details about the knowledge and passion you bring to the subject(s) you’d like to teach. Providers look for evidence of both interest and understanding.</p>
-  <p class="govuk-body">Check your provider’s course listing page on Find postgraduate teacher training for specific subject knowledge requirements.</p>
+  <p class="govuk-body-l">Use this section to give evidence of the knowledge and interest you bring to the subject(s) you'd like to teach.</p>
+  <p class="govuk-body">This is particularly important if your choice of subject is different from that of your degree. Check your provider’s course listing page on <a href=\"https://find-postgraduate-teacher-training.education.gov.uk/?utm_source=getintoteaching&utm_medium=referral&utm_content=trainingtoteachbizstudies\">Find postgraduate teacher training</a> for their specific subject knowledge requirements.</p>
```

Comment:
Should remove UTM codes as 1.) details included in query likely incorrect 2.) its needless tracking, certainly for a prototype.

```suggestion
  <p class="govuk-body">This is particularly important if your choice of subject is different from that of your degree. Check your provider’s course listing page on <a href=\"https://find-postgraduate-teacher-training.education.gov.uk/">Find postgraduate teacher training</a> for their specific subject knowledge requirements.</p>
```